index.js

```diff
@@ -62,14 +39,24 @@ module.exports = {
       trees.push(tree);
     }
 
-    let materializePath = path.join(this.project.root, this.app.bowerDirectory, 'materialize', 'dist', 'js');
+    let materializePath = path.join(MATERIALIZE_JS_PATH, '..');
     if (existsSync(materializePath)) {
-      let materializeTree = fastbootTransform(new Funnel(materializePath, {
-        files: ['materialize.js'],
-        destDir: 'materialize'
-      }));
+      let materializeJsTree = fastbootTransform(
+        new Funnel(materializePath, {
+          files: ['materialize.js'],
+          destDir: 'materialize'
+        })
+      );
+      let materializeFontsTree = fastbootTransform(
+        new Funnel(path.join(materializePath, '..', 'fonts'), {
+          destDir: 'fonts'
+        })
+      );
```

Comment:
Technically we don't need to refrain from dealing with fonts in fastboot mode, but there's no benefit either.

Reply:
Fonts are going to need some changes with 1.x since they [won't be shipping with materialize](https://github.com/Dogfalo/materialize/commit/3cb0791de80edbff33ee4a53aa422d4ff2af6149) any more. We could pull them from [here](https://github.com/choffmeister/roboto-fontface-bower) [though](https://www.npmjs.com/package/roboto-fontface).
BTW, do you think we should make fonts optional just like with `.omitJs`?